decentraland/unity-explorer
# Fix: local videos playback on sdk7 scenes

## What does this PR change?

- add support for local scene videos via getting proper mediaUrl for it (similar to what we do with Textures)

...

## How to test the changes?

1. Launch the explorer
2. /goto -3,133
3. verify you see the blue video with digits on the screens
![image](https://github.com/decentraland/unity-explorer/assets/35366872/4b5e2c2c-57e0-4144-8728-4e5d890e86aa)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
